### DISCUSSION TITLE
Ruby - Command failed: ./local/bin/exec lets lint Hatası

### DISCUSSION THREAD

**Discussion by aenesyvz** [19/classroom#37] [Anonim] [answered]
![Ekran Alıntısı](https://user-images.githubusercontent.com/72816920/158902932-6d4b96d7-5613-4d38-a77b-998d5ca32e65.PNG)

Merhabalar, terminalde "lets grad" komutunu çalıştırdığımda Good değerlendirme kısmında "Command failed : ./local/bin/exec lets lint" hatası alıyorum. Hatanın çözümü için birçok dokümantasyon okusam da hatayı çözemedim.  rubocop'u çalıştırdığımda ise dosya isminde snake_case kullanımı ve class Point için yorum satırı eklemek dışında hiçbir hata gözükmüyor. rubocop' un verdiği bu hataları göz ardı edebileceğimiz bir önceki tartışmada söylenilmiş. point-distance.rb dosyası hariç hiçbir dosyada değişiklik yapmadım. Değerlendirmeler sonucu 80 puan alıyorum. Bu sorunu nasıl çözebilirim? Saygılarımla.

**Comment by roktas** [comment, marked as answer]
Gördüğüm kadarıyla Classroom ortamını başarıyla kurmuşsun.  Yukarıda ekran görüntüsünü verdiğin terminalde (VS Code ile açtığın terminal)  şu komutları verip tekrar dener misin?

```sh
sudo classroom -verbose renew
lets-grad
```

**Reply by aenesyvz** [reply]
Çok teşekkür ederim hocam. Sorunum çözüldü.

**Reply by roktas** [reply]
Arada düzeltmeler yayınlıyorum.  Kurulumun güncel değildi, nedeni bu.
